backend/infrahub/core/migrations/graph/m018_uniqueness_nulls.py

```diff
@@ -95,5 +96,5 @@ class Migration018(InternalSchemaMigration):
     async def validate_migration(self, db: InfrahubDatabase) -> MigrationResult:  # noqa: ARG002
         return MigrationResult()
 
-    async def execute(self, db: InfrahubDatabase) -> MigrationResult:
+    async def execute(self, db: InfrahubDatabase, user_id: str = SYSTEM_USER_ID) -> MigrationResult:  # noqa: ARG002
```

Comment:
Not for now but with this in mind and for future changes we could consider having some upgrade package that we include as an argument to execute where the database would be one attribute and the user_id would be an optional one would also take care of the ignore to the type.